examples/speaker_tasks/diarization/conf/msdd_5scl_15_05_50Povl_256x3x32x2.yaml

```diff
@@ -0,0 +1,126 @@
+# Mutiscale diarization decoder (MSDD) is a speaker diarization model based on initializing clustering and multi-scale segmentation input.
```

Comment:
can you provide a sample json line of train manifest

Reply:
Sure added